starters/next-wordpress-starter/components/grid.js

```diff
@@ -58,3 +58,37 @@ export const PostGridItem = ({ content: post }) => {
     </Link>
   );
 };
+
+export const PageGridItem = ({ content: page }) => {
+  const imgSrc = page?.featuredImage?.node.sourceUrl || "";
+  const altText = page?.featuredImage?.node.altText || page.title;
+
+  return (
+    <Link passHref href={`/page${page.uri}`}>
```

Comment:
I think we'll want the prefix here to be `/pages`

Reply:
Nice catch!